distribution/ddf-common/src/main/resources/etc/definitions/ddf-catalog-taxonomy-injectors.json

```diff
@@ -0,0 +1,250 @@
+{
+  "inject": [
+    {
+      "attribute": "associations.external"
+    },
+    {
+      "attribute": "metacard.associations.related"
+    },
+    {
+      "attribute": "metacard.associations.derived"
+    },
+    {
+      "attribute": "contact.contributor-address"
+    },
+    {
+      "attribute": "contact.contributor-email"
+    },
+    {
+      "attribute": "contact.contributor-name"
+    },
+    {
+      "attribute": "contact.contributor-phone"
+    },
+    {
+      "attribute": "contact.creator-address"
+    },
+    {
+      "attribute": "contact.creator-email"
+    },
+    {
+      "attribute": "contact.creator-name"
+    },
+    {
+      "attribute": "contact.creator-phone"
+    },
+    {
+      "attribute": "contact.point-of-contact-address"
+    },
+    {
+      "attribute": "contact.point-of-contact-email"
+    },
+    {
+      "attribute": "contact.point-of-contact-name"
+    },
+    {
+      "attribute": "contact.point-of-contact-phone"
+    },
+    {
+      "attribute": "contact.publisher-address"
+    },
+    {
+      "attribute": "contact.publisher-email"
+    },
+    {
+      "attribute": "contact.publisher-name"
+    },
+    {
+      "attribute": "contact.publisher-phone"
+    },
+    {
+      "attribute": "datetime.start"
+    },
+    {
+      "attribute": "datetime.name"
+    },
+    {
+      "attribute": "datetime.end"
+    },
+    {
+      "attribute": "location.altitude-meters"
+    },
+    {
+      "attribute": "location.country-code"
+    },
+    {
+      "attribute": "location.crs-code"
+    },
+    {
+      "attribute": "location.crs-name"
+    },
+    {
+      "attribute": "media.format"
```

Comment:
❓ Are media attributes really core to the taxonomy of every metacard? I'm just thinking of all the metacards that will never use them and am curious why these are considered core (They seem very resource specific)

Reply:
They are considered part of the common catalog taxonomy I thought.

Reply:
Not sure if this is a good authority (or even up-to-date), but may have some answers to what is part of the common catalog taxonomy.
https://codice.atlassian.net/wiki/spaces/DDF/pages/36995079/Attributes

Reply:
That's not up-to-date. The most authoritative source is the asciidoc.

Reply:
Do we really consider the media attributes to generic to all metacards though?? It would make sense in the legacy sense where everything is a resource, but now days a resource is not the most generic unit of data.

Reply:
@codice-bot It should, whoever you are. 😛

Reply:
Was able to get @cxbot to assist with the removal of the wikki pages